### PR TITLE
[PHP] minor code enhancement

### PR DESCRIPTION
**Pull request by wing328**
- minor code enhancement (better cast)
- remove trailing spaces

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `5.1.x`, `6.0.x`
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

cc @jebentier (2017/07), @dkarlovi (2017/07), @mandrean (2017/08), @jfastnacht (2017/09), @ackintosh (2017/09) ❤️, @ybelenko (2018/07), @renepardon (2018/12)

### CODE REVIEW

**Comment by ybelenko** [comment]
Tested locally

```console
$  vendor/bin/phpunit tests
PHPUnit 9.4.0 by Sebastian Bergmann and contributors.

Warning:       Your XML configuration validates against a deprecated schema.
Suggestion:    Migrate your XML configuration using "--migrate-configuration"!

EEEEE.........ES.................................ESSSSSSSSSSSSSS. 65 / 73 ( 89%)
......E.                                                          73 / 73 (100%)

Time: 00:09.095, Memory: 10.00 MB

There were 8 errors:

1) OpenAPI\Client\AsyncTest::testAsyncRequest
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>


/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:134
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/AsyncTest.php:40

2) OpenAPI\Client\AsyncTest::testAsyncRequestWithHttpInfo
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>


/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:134
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/AsyncTest.php:40

3) OpenAPI\Client\AsyncTest::testAsyncThrowingException
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>


/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:134
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/AsyncTest.php:40

4) OpenAPI\Client\AsyncTest::testAsyncApiExceptionWithoutWaitIsNotThrown
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>


/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:134
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/AsyncTest.php:40

5) OpenAPI\Client\AsyncTest::testAsyncHttpInfoThrowingException
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>


/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:134
/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/AsyncTest.php:40

6) OpenAPI\Client\DebugTest::testEnableDebugOutput
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>
 in /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
Stack trace:
#0 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/DebugTest.php(15): OpenAPI\Client\Api\PetApi->addPetWithHttpInfo(Object(OpenAPI\Client\Model\Pet))
#1 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/Framework/TestSuite.php(614): OpenAPI\Client\DebugTest::setUpBeforeClass()
#2 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/Framework/TestSuite.php(669): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#3 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#4 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/Command.php(148): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#5 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/Command.php(101): PHPUnit\TextUI\Command->run(Array, true)
#6 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/phpunit(61): PHPUnit\TextUI\Command::main()
#7 {main}
7) OpenAPI\Client\PetApiTest::testGetPetById
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>
 in /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
Stack trace:
#0 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/PetApiTest.php(49): OpenAPI\Client\Api\PetApi->addPetWithHttpInfo(Object(OpenAPI\Client\Model\Pet))
#1 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/Framework/TestSuite.php(614): OpenAPI\Client\PetApiTest::setUpBeforeClass()
#2 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/Framework/TestSuite.php(669): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#3 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#4 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/Command.php(148): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#5 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/Command.php(101): PHPUnit\TextUI\Command->run(Array, true)
#6 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/phpunit(61): PHPUnit\TextUI\Command::main()
#7 {main}
8) OpenAPI\Client\StoreApiTest::testGetInventory
OpenAPI\Client\ApiException: [405] Client error: `POST http://petstore.swagger.io/v2/pet` resulted in a `405 Method Not Allowed` response:
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><apiResponse><type>unknown</type></apiResponse>
 in /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php:161
Stack trace:
#0 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php(134): OpenAPI\Client\Api\PetApi->addPetWithHttpInfo(Object(OpenAPI\Client\Model\Pet))
#1 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/tests/StoreApiTest.php(47): OpenAPI\Client\Api\PetApi->addPet(Object(OpenAPI\Client\Model\Pet))
#2 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/Framework/TestSuite.php(614): OpenAPI\Client\StoreApiTest::setUpBeforeClass()
#3 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/Framework/TestSuite.php(669): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#4 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(673): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#5 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/Command.php(148): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\Framework\TestSuite), Array, Array, true)
#6 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/src/TextUI/Command.php(101): PHPUnit\TextUI\Command->run(Array, true)
#7 /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/vendor/phpunit/phpunit/phpunit(61): PHPUnit\TextUI\Command::main()
#8 {main}
ERRORS!
Tests: 73, Assertions: 149, Errors: 8, Skipped: 15.
```

```console
$  vendor/bin/php-cs-fixer fix --allow-risky=yes
Loaded config default from "/Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/.php_cs".
Using cache file ".php_cs.cache".

Fixed all files in 0.319 seconds, 22.000 MB memory used

Files that were not fixed due to errors reported during linting before fixing:
   1) /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Model/OuterEnumIntegerDefaultValue.php
   2) /Users/ybelenko/Sites/openapi-generator/samples/client/petstore/php/OpenAPIClient-php/lib/Model/OuterEnumInteger.php
```

File: modules/openapi-generator/src/main/resources/php/api.mustache [hunk 1]
```diff
@@ -187,7 +187,7 @@ use {{invokerPackage}}\ObjectSerializer;
             } catch (RequestException $e) {
                 throw new ApiException(
                     "[{$e->getCode()}] {$e->getMessage()}",
-                    $e->getCode(),
+                    (int) $e->getCode(),
```
Comment (ybelenko) [inline modules/openapi-generator/src/main/resources/php/api.mustache:190]: Cannot remember which linter(PHPStan or PHP_CodeSniffer) translates:
```diff
--- (int) $e->getCode(),
+++ (int)$e->getCode(),
```
But current [PSR-12 example](https://www.php-fig.org/psr/psr-12/#61-unary-operators) writes type casting with space :
> Type casting operators MUST NOT have any space within the parentheses
> `$intValue = (int) $input;`

So let's consider both coding styles correct. The only thing that current PSR declares is no space within parentheses.

**Review by ybelenko — APPROVED** [review]

**Comment by wing328** [comment]
About the test failure, you will need to run the petstore test server locally.

**Comment by ybelenko** [comment]
> About the test failure, you will need to run the petstore test server locally.

Yes, I remember about local test server. I saved scripts output here just to show that I launched it locally and in case it may be useful in future. Current CI doesn't run these tasks if I remember correct, at least last one with PHPCSFixer.

I'm totally agree with PR merge.

**Pull request merged**